build_tools/cmake/iree_copts.cmake

```diff
@@ -412,18 +429,6 @@ if(IREE_SIZE_OPTIMIZED)
   )
 endif()
 
-#-------------------------------------------------------------------------------
```

Comment:
Actually, is this related? Or would we want even other options here still?

Reply:
It was originally related as I was initially adding the new code in this part of the file, so I removed this unneeded placeholder.  I didn't bother to restore it, it seemed to have outlived its purpose.

Reply:
I'll wait for another approval from either @GMNGeoffrey, @ScottTodd or @pzread before going ahead with the merge.

Reply:
I'd keep those lines.